### DISCUSSION TITLE
Полностью "удаленная" работа по проекту

### DISCUSSION THREAD

**Discussion by nergal-perm** [nergal-perm/tg-core#27] [Закрытые обсуждения] [answered]
Что, если попробовать вообще все взаимодействие участников проекта свести к задачам, обсуждениям, статьям в wiki и исходному коду в этом репозитории? 

Если нужно, чтобы кто-то что-то сделал - создай ему задачу. Если есть новая идея - оставь сообщение на форуме. Обсудили идею и детали реализации в комментариях - зафиксировали на соответствующей странице в wiki.

Личные встречи и обсуждения голосом обязательно протоколировать с максимумом подробностей.

У такого подхода есть серьезное достоинство с побочным эффектом - работа над проектом перестает зависеть от часовых поясов и расписаний отдельных участников, и к тому же в идеале остается полная и подробная документация.

**Comment by citerm** [comment, marked as answer]
Полностью поддерживаю, но думаю что использовать сразу все инструменты неудобно. Не понятно как из одной сущности возникает другая, за ней следующая и т.д. Нам нужен флоу, о котором мы договоримся.

**Reply by citerm** [reply]
Я попробую сделать схемку в miro, и мини-описание в wiki
Предварительно это: Discussion -> Wiki Page(s) + Miro -> Issue(s)
Discussion - обсуждение бизнес требований, функциональных требований, регламента работы
Wiki Page - фиксация договоренностей (бизнес, функциональные, код, регламенты) и ссылки на схемы в Miro
Issue - задачи для разработки, код и тестирование

**Reply by citerm** [reply]
Схемка https://miro.com/app/board/o9J_kwSPgS0=/?moveToWidget=3074457355111347752&cot=14

**Reply by citerm** [reply]
Сделал вот такую страничку [Регламент совместной работы ...](https://github.com/nergal-perm/tg-core/wiki/%D0%A0%D0%B5%D0%B3%D0%BB%D0%B0%D0%BC%D0%B5%D0%BD%D1%82-%D1%81%D0%BE%D0%B2%D0%BC%D0%B5%D1%81%D1%82%D0%BD%D0%BE%D0%B9-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%8B-%D1%81-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5%D0%BC-%D0%B2%D1%81%D1%82%D1%80%D0%BE%D0%B5%D0%BD%D0%BD%D1%8B%D1%85-%D0%B8%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2-GitHub)

**Reply by citerm** [reply]
Чувствую, что пора завести дискуссию для обсуждения игры #31

**Reply by nergal-perm** [reply]
Полностью согласен и со схемой, и с текстовым описанием в wiki.

**Comment by nergal-perm** [comment]
На данном этапе о рабочем процессе мы договорились.

- [Описание в wiki](https://github.com/nergal-perm/tg-core/wiki/%D0%A0%D0%B5%D0%B3%D0%BB%D0%B0%D0%BC%D0%B5%D0%BD%D1%82-%D1%81%D0%BE%D0%B2%D0%BC%D0%B5%D1%81%D1%82%D0%BD%D0%BE%D0%B9-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%8B-%D1%81-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5%D0%BC-%D0%B2%D1%81%D1%82%D1%80%D0%BE%D0%B5%D0%BD%D0%BD%D1%8B%D1%85-%D0%B8%D0%BD%D1%81%D1%82%D1%80%D1%83%D0%BC%D0%B5%D0%BD%D1%82%D0%BE%D0%B2-GitHub).
- [Схема в Miro](https://miro.com/app/board/o9J_kwSPgS0=/?moveToWidget=3074457355111347752&cot=14) 

Дискуссию можно считать законченной. 

Одна проблема - Github не позволяет окончательно закрывать дискуссии, все активные участники репозитория могут без ограничений продолжать добавлять комменты. Поэтому я создал новый вопрос на обсуждение того, каким образом отделять закрытые дискусии от открытых #32.
